content/posts/whats-new-august.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: What's New With Zoo, August Edition
+excerpt: All the latest updates from the past month
+date: '2025-08-05'
+author:
+    name: Adam Chalmers
+    picture: '/adamchalmers.jpg'
+--- 
+
+Hi, Zoo users! This is the first of our planned monthly blog posts highlighting all the improvements we’ve made to Zoo each month. We’ve been hard at work fixing bugs, improving performance and adding features since 1.0, and we’re excited to share them with you.
+
+---
+
+## Much improved Text-to-CAD reliability
+
+When we first launched Text-to-CAD in December 2023, it could only create a single CAD file, like a STEP or OBJ file that contained a 2ft wide bookshelf. This was fine, but users kept asking for a more powerful Text-to-CAD that could create parametric models, e.g. a bookshelf with any width. When we launched Zoo Design Studio 1.0 in May, it came with built-in parametric Text-to-CAD. You enter a text prompt, and our model spits out parametric KCL, the KittyCAD programming language. This meant you would get a bookshelf where the bookshelf's width is assigned once to a variable. If you change that variable later, all the bookshelf's other measurements will recalculate themselves using the new width. This is much better than constantly designing the same bookshelf again and again with different measurements.
+
+When we replaced the simple Text-to-CAD model with our new parametric model, we removed the old Text-to-CAD website and instead moved all our ML tooling into Zoo Design Studio. That annoyed a lot of our old users, so we recently put it back. You can now visit text-to-cad.zoo.dev to get the old Text-to-CAD experience back.
+
+Another problem was that our new parametric model was having trouble generating KCL. Although the new model was much more powerful, it wasn't as reliable. Improving its KCL generation has been our #1 priority for the last few months, and we're proud to report that we've reduced the error rate from 50% to 16%. 
+
+
+![Error chart showing errors dropping and success rising](ml_errors.png)
+
+This has been a focus for multiple teams at Zoo, including the ML team, the KCL team, and the engine (CAD kernel) team. We'll be going over some of the specific improvements in a following blog post, and we're hard at work lowering the error rate even further.
```

Comment:
Maybe we put something like "Errors are just part of the story here, once we overcome that hurdle we will have more time to focus on making sure the model is what you asked for."

idk or something in your own words just so people arent like errors dropped but now i get turds